app/components/agenda-manager/agenda-item-form/select-draft.js

```diff
@@ -17,6 +17,10 @@ export default class AgendaManagerAgendaItemFormSelectDraftComponent extends Com
     this.getDrafts.perform();
   }
 
+  get agendaItem() {
+    return this.args.model;
```

Comment:
I aliased it since I think using "model" is confusing, but it's just a quick solution, I think it would be better if it was just passed in with a better name to be honest.

Reply:
I'm definitely OK with renaming the argument!